src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/ImageTests/Image_Margins.xaml.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Windows.UI.Xaml.Controls;
+using Uno.UI.Samples.Controls;
+using Windows.UI.Xaml;
+using System;
+using System.Threading.Tasks;
+using UIKit;
+
+namespace UITests.Windows_UI_Xaml_Controls.ImageTests
+{
+	[SampleControlInfo(category: "Image")]
+	public sealed partial class Image_Margins : Page
+	{
+		public Image_Margins()
+		{
+			this.InitializeComponent();
+
+			Loaded += L;
+		}
+
+		private async void L(object sender, RoutedEventArgs e)
+		{
+			await Task.Delay(200);
+			//tree.Text = this.ShowLocalVisualTree();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5198510)